metadata/metadata.go

```diff
@@ -14,19 +14,28 @@
 
 package metadata
 
-import "errors"
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/mendersoftware/log"
+)
+
+type Validater interface {
```

Comment:
Ugh, that's a bizarre requirement, it enforces spelling errors! But if that's how it is, we have no choice I guess.